ClientRuntimes/CSharp/ClientRuntime.Azure/JsonSerializerExtensions.cs

```diff
@@ -24,8 +24,8 @@ public static class JsonSerializerExtensions
                 throw new ArgumentNullException("serializer");
             }
             JsonSerializer newSerializer = new JsonSerializer();
-            PropertyInfo[] properties = typeof(JsonSerializer).GetProperties();
-            foreach (var property in properties.Where(p => p.GetSetMethod() != null))
+            var properties = typeof(JsonSerializer).GetTypeInfo().DeclaredProperties;
```

Comment:
This is using the .net 4.5-based reflection library instead of the .net 40 based relfection library